dev-setup/debian-setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# First we should go home
+cd ~/
+
+# Add PostgreSQL's repository to your system:
+echo "Adding PostgreSQL's repository to your system..."
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+
+# Import the repository signing key:
+echo "Importing the PostgreSQL's repository signing key..."
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+# Update the package lists:
+echo "Updating package lists..."
+sudo apt update 
+
+# Install Rust build tools and dependencies:
+echo "Installing development tools and dependencies..."
+sudo apt install -y build-essential libssl-dev pkg-config gettext gcc g++ curl git
+
+# Install PostgreSQL:
+echo "Installing PostgreSQL..."
+sudo apt install postgresql libpq-dev
+
+# Install Rust:
+echo "Installing Rust..."
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Install Rust toolchain:
+echo "Installing Rust stable toolchain..."
+rustup install stable
+
+# Install Rust tools:
+echo "Installing Rust tools..."
+rustup component add rustfmt clippy cargo-watch
+cargo install diesel_cli --no-default-features --features "postgres"
+
+# Clone Aardwolf:
+echo "Switching to home directory"
+cd ~/
+echo "Cloning Aardwolf..."
+git clone https://github.com/aardwolf/aardwolf
+
+# Setup PostgreSQL
+echo "Setting up PostgreSQL..."
+sudo systemctl enable postgresql.service
+sudo systemctl start postgresql.service
+
+# Create the aardwolf database
+echo "Creating the aardwolf database..."
+sudo -u postgres psql -c "CREATE DATABASE aardwolf;"
+sudo -u postgres psql -c "CREATE DATABASE aardwolf_testing;"
+
+# Create the aardwolf database user
+echo "Creating the aardwolf database user..."
+sudo -u postgres psql -c "CREATE USER aardwolf_user WITH PASSWORD 'changeme';"
```

Comment:
**Suggestion:** Consider using a variable for the database name, user, and password. This will make it easier to change these values in the future.
```suggestion
DB_NAME=aardwolf
DB_USER=aardwolf_user
DB_PASS=changeme

sudo -u postgres psql -c "CREATE DATABASE $DB_NAME;"
sudo -u postgres psql -c "CREATE DATABASE ${DB_NAME}_testing;"
sudo -u postgres psql -c "CREATE USER $DB_USER WITH PASSWORD '$DB_PASS';"
```